xscen/catalog.py

```diff
@@ -585,15 +618,15 @@ def __init__(
 
         Parameters
         ----------
-        df : str, dict
+        df : str or dict
             If str, this must be a path or URL to a catalog JSON file.
             If dict, this must be a dict representation of an ESM catalog.  See the notes below.
         create : bool
             If True, and if 'df' is a string, this will create an empty ProjectCatalog if none already exists.
-        project : dict-like
-            Metadata to create the catalog, if required.
-        overwrite : bool
+        overwrite : bool, optional
```

Comment:
Why not set this to `False` by default? This means it accepts `True | False | None`.

Reply:
Indeed, this made no sense. I removed the option to put `None` here (especially since it wasn't actually supported by `ProjectCatalog.create()`.